README.md

```diff
@@ -42,12 +42,13 @@ See [Testing Farm onboarding guide](https://docs.testing-farm.io/Testing%20Farm/
 
 ### Test Environment
 
-| Input Name | Description | Default value |
-|------------|-------------|---------------|
-| `compose` | Compose to run tests on. [Available composes.](https://api.dev.testing-farm.io/v0.1/composes) | Fedora-latest |
-| `arch` | Define an architecture for testing environment | x86_64 |
-| `variables` | Environment variables for test env, separated by ; | empty |
-| `secrets` | Environment secrets for test env, separated by ; | empty |
+| Input Name  | Description                                                                                    | Default value |
```

Comment:
Please do not make all the rows the same length. The git history is then harder to read.

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/146/commits/3d807fe8f0e1940a6d7dc4901641eb18e190f868